app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/nutrition/NutritionProductFragment.java

```diff
@@ -284,6 +291,22 @@ public void refreshView(State state) {
         rv.setLayoutManager(new LinearLayoutManager(getContext()));
         rv.setAdapter(new NutrientLevelListAdapter(getContext(), levelItem));
 
+        textNutriScoreInfo.setClickable(true);
+        textNutriScoreInfo.setMovementMethod(LinkMovementMethod.getInstance());
+        SpannableStringBuilder spannableStringBuilder=new SpannableStringBuilder();
+        ClickableSpan clickableSpan=new ClickableSpan() {
+            @Override
+            public void onClick(@NonNull View view) {
+                CustomTabsIntent customTabsIntent = new CustomTabsIntent.Builder().build();
+                customTabsIntent.intent.putExtra("android.intent.extra.REFERRER", Uri.parse("android-app://" + getActivity().getPackageName()));
+                CustomTabActivityHelper.openCustomTab(getActivity(), customTabsIntent, Uri.parse("http://world.openfoodfacts.org/nutrient-levels"), new WebViewFallback());
```

Comment:
Can you extract the URL to a translatable string ?

Reply:
yeah okay